pkg/pkg.mk

```diff
@@ -82,8 +82,15 @@ PKG_CUSTOM_PREPARED ?=
 # Declare 'all' first to have it being the default target
 all: prepare
 
+BUILD_DIR ?= $(RIOTBASE)/build
```

Comment:
I feel this recipe should be placed outside of this file, as it is at a broader scope than building packages. Perhaps as an include-able makefile in `./makefiles`. However, I will approve as-is.